resolwe_bio/processes/alignment/bowtie.yml

```diff
@@ -278,12 +278,14 @@
       re-save species {{genome.species}}
       re-save build {{genome.build}}
 
-      bamtobigwig.sh "${FW_NAME}_mapped.bam" {{ requirements.resources.cores }} 50
-
       if [ -f "${FW_NAME}_unmapped.bam" ]; then
         re-save-file unmapped "${FW_NAME}_unmapped.fastq.gz"
       fi
 
+      #allowed time for bamCoverage is 480s
```

Comment:
Improve this comment, e.g. computation time limit for bamCoverage is 480 s per GB. Do the same in other processes.